package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@grafana/experimental",
-  "version": "0.0.2-canary.22",
+  "version": "0.0.2-canary.23",
```

Comment:
How come this yelds a diff? version on main is .23 already...

Reply:
Maybe try rebasing it

Reply:
Super weird, rebasing it dropped the change. I'll close this.